API Tokens (#5146)
ckan/ckan: doc/api/index.rst

do something in the web interface they'll be authorized to do it via the API as
well.

When calling an API function that requires authorization, you must
authenticate yourself by providing your API key or API token with your
HTTP request. To find your API key, login to the CKAN site using its

> web interface and visit your user profile page. To generate new API
> token visit **API Tokens** tab available from your puser profile's

Reviewer: ```suggestion
web interface and visit your user profile page. To generate a new API
token visit the *API Tokens* tab available on your user profile.
```

Also can you rename this section to "Authentication, API keys and API tokens" and replace "(Replacing XXX with your API key.)" with "(Replacing XXX with your API key or token.)"

If we want to encourage people to use them and deprecate the API key we should make them more prominent in this version docs.
Is it worth putting a `..note::`  saying that API keys will get deprecated/removed in the next version?

Author: Done. I've added a note about deprecation of api keys. I don't think that we'll really remove them before 3.1/3.2, so the note is quite vague:)
